implementation/extract-message-filter/bin/extract-message-filter.ts

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import { ExtractMessageFilterOriginalStack } from '../lib/extract-message-filter-original';
+import { ExtractMessageFilterRefactoredStack } from '../lib/extract-message-filter-refactored';
+import { AwsSolutionsChecks, NagSuppressions } from 'cdk-nag'
+import { Aspects } from 'aws-cdk-lib';
+
+const app = new cdk.App();
+Aspects.of(app).add(new AwsSolutionsChecks({ verbose: true }))
+
+const originalStack = new ExtractMessageFilterOriginalStack(app, 'ExtractMessageFilterOriginalStack', {});
+const refactoredStack = new ExtractMessageFilterRefactoredStack(app, 'ExtractMessageFilterRefactoredStack', {})
+
+const NagSupressionList = [
+    { id: 'AwsSolutions-S1', reason: 'S3 bucket is for demo purposes, not enabling server access logs' },
+    { id: 'AwsSolutions-S2', reason: 'S3 bucket is for demo purposes, skipping public access restriction' },
+    { id: 'AwsSolutions-S10', reason: 'S3 bucket is for demo purposes, skipping SSL requirement' },
+    { id: 'AwsSolutions-IAM4', reason: 'This is a demo application, allowing use of AWS Managed policies' },
```

Comment:
Our infosec does not allow AWS Managed policies even when it is a demo application. Checkout our other examples, say send-message, on how to replace AWS managed policies with custom managed policies.